python_src/src/lamp_py/performance_manager/l0_rt_vehicle_positions.py

```diff
@@ -101,97 +105,6 @@ def transform_vp_datatypes(
     return vehicle_positions
 
 
-def join_vp_with_gtfs_static(
-    vehicle_positions: pandas.DataFrame, db_manager: DatabaseManager
-) -> pandas.DataFrame:
-    """
-    join vehicle position dataframe to gtfs static records
-
-    adds "fk_static_timestamp" and "parent_station" columns to dataframe
-    """
-    process_logger = ProcessLogger(
-        "vp.join_with_gtfs_static", row_count=vehicle_positions.shape[0]
-    )
-    process_logger.log_start()
-
-    # get unique "start_date" values from vehicle position dataframe
-    # with associated minimum "vehicle_timestamp"
-    date_groups = vehicle_positions.groupby(by="start_date")[
-        "vehicle_timestamp"
-    ].min()
-
-    # dictionary used to match minimum "vehicle_timestamp" values to
-    # "timestamp" from StaticFeedInfo table, to be used as foreign key
-    timestamp_lookup = {}
```

Comment:
Broke this function up into two utility functions in `gtfs_utils.py` since it is used by Vehicle Positions and Trip Updates processing. This one function is replaced by `add_fk_static_timestamp_column` and `add_parent_station_column`